plugins/redfish/fu-redfish-device.c

```diff
@@ -360,6 +360,26 @@ fu_redfish_device_set_vendor(FuRedfishDevice *self, const gchar *vendor)
 	fu_device_add_vendor_id(FU_DEVICE(self), vendor_id);
 }
 
+static void
+fu_redfish_backend_smc_license_check(FuDevice *device)
+{
+	FuRedfishDevice *self = FU_REDFISH_DEVICE(device);
+	FuRedfishBackend *backend = fu_redfish_device_get_backend(self);
+	g_autoptr(FuRedfishRequest) request = fu_redfish_backend_request_new(backend);
+	g_autoptr(GError) error_local = NULL;
+
+	/* see if we don't get an license error */
+	if (!fu_redfish_request_perform(request,
+					fu_redfish_backend_get_push_uri_path(backend),
+					FU_REDFISH_REQUEST_PERFORM_FLAG_LOAD_JSON,
+					&error_local)) {
+		if (g_error_matches(error_local, FWUPD_ERROR, FWUPD_ERROR_NOT_SUPPORTED))
+			fu_device_add_problem(device, FWUPD_DEVICE_PROBLEM_MISSING_LICENSE);
```

Comment:
For debugging purpose would it make sense to output the error the debug log if it _doesn't match_ here?